embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs

```diff
@@ -0,0 +1,360 @@
+                byte[] bytes = Encoding.UTF8.GetBytes(value);
+                byte[] encodedValue = codec.EncodeMessage(bytes);
+                byte[] expected;
+                using (var stream = new MemoryStream())
+                {
+                    stream.WriteByte(BYTE_ARRAY);
+                    CodecTestsHelper.WriteBytes(stream, bytes);
+                    expected = stream.ToArray();
+                }
+                Assert.Equal(expected, encodedValue);
+            }
+
+            [Theory]
+            [InlineData(new int[] { 1, 2, 5 }, INT_ARRAY, 4)]
+            [InlineData(new long[] { 1L, 2L, 5L }, LONG_ARRAY, 8)]
+            [InlineData(new float[] { 1.1f, 2.2f, 5.5f }, FLOAT_ARRAY, 4)]
+            [InlineData(new double[] { 1.1d, 2.2d, 5.5d }, DOUBLE_ARRAY, 8)]
+            public void Encodes_Correct_ArrayTypes(ICollection content, byte type, int alignment)
+            {
+                var codec = StandardMessageCodec.Instance;
+                byte[] encodedValue = codec.EncodeMessage(content);
+                byte[] expected;
+                using (var stream = new MemoryStream())
+                {
+                    stream.WriteByte(type);
+                    CodecTestsHelper.WriteSize(stream, content.Count);
+                    CodecTestsHelper.WriteAlignment(stream, alignment);
+                    foreach (dynamic value in content)
+                    {
+                        stream.Write(BitConverter.GetBytes(value));
+                    }
+                    expected = stream.ToArray();
+                }
+                Assert.Equal(expected, encodedValue);
+            }
+
+            [Fact]
+            public void Encodes_Correct_String()
+            {
+                var codec = StandardMessageCodec.Instance;
+                string content = StringCodecTests.TEST_STRING;
+                byte[] encodedValue = codec.EncodeMessage(content);
+                byte[] expected;
+                using (var stream = new MemoryStream())
+                {
+                    stream.WriteByte(STRING);
+                    CodecTestsHelper.WriteBytes(stream, StringCodecTests.TEST_BYTES);
+                    expected = stream.ToArray();
+                }
+                Assert.Equal(expected, encodedValue);
+            }
+
+            [Fact]
+            public void Encodes_Correct_Map()
+            {
+                var codec = StandardMessageCodec.Instance;
+                var content = new Dictionary<string, object>
+                {
+                    ["Key_1"] = 1,
+                    ["Key_2"] = "TEST_STRING",
+                    ["Key_3"] = true
+                };
+                byte[] encodedValue = codec.EncodeMessage(content);
+                byte[] expected;
+                using (var stream = new MemoryStream())
+                {
+                    stream.WriteByte(MAP);
+                    CodecTestsHelper.WriteSize(stream, 3);
```

Comment:
It doesn't matter but I just noticed the mixed use of `stream.WriteByte(3)` and `CodecTestsHelper.WriteSize(stream, 3)`.

Reply:
the helper class is needed to simplify code. I added `StandardMessageHelper` class and modified the code more simply.

Reply:
What I meant was you used `stream.WriteByte(3)` to specify the size of LIST and `WriteSize(stream, 3)` for MAP. Just a small consistency problem.

Reply:
Ah I see. I fixed them.